tests/test_NFSSR.py

```diff
@@ -89,3 +89,8 @@ def test_attach(self, validate_nfsversion, check_server_tcp, _testhost,
                                            timeout=100,
                                            nfsversion='aNfsversionChanged',
                                            retrans=3)
+
+    @mock.patch('NFSSR.Lock', autospec=True)
+    def test_load_ipv6(self, Lock):
```

Comment:
make this `lock` or `mock_lock`, CamelCase isn't PEP compliant syntax

Reply:
`Lock` is used in all other tests with capital L, won't fail if I change it? I'll try.

Reply:
I'm only saying change the parameter to `test_load_ipv6` that's just the mock object that is created by the `@mock.patch` call so that you could if you needed setup up some pre-conditions or responses and check the thing got called as you expect. It doesn't change what the code under test is called.
i.e.
```
@mock.patch('NFSSR.Lock', autospec=True)
    def test_load_ipv6(self, mock_lock):
```